tests/components/statistics/test_sensor.py

```diff
@@ -920,6 +921,24 @@ async def test_state_characteristics(hass: HomeAssistant) -> None:
             "value_9": float(round(sum(VALUES_NUMERIC) / len(VALUES_NUMERIC), 2)),
             "unit": "°C",
         },
+        {
+            "source_sensor_domain": "sensor",
+            "name": "mean_circular",
+            "value_0": STATE_UNKNOWN,
+            "value_1": float(VALUES_NUMERIC[-1]),
+            "value_9": float(
+                round(
+                    math.degrees(
+                        math.atan2(
+                            sum(math.sin(math.radians(x)) for x in VALUES_NUMERIC),
+                            sum(math.cos(math.radians(x)) for x in VALUES_NUMERIC),
+                        )
+                    ),
+                    2,
+                )
```

Comment:
Hey, could you replace the formula by the actual result, or some simpler representation of it? When the test case uses the exact same logic as the main function, it's a self fulfilling promise :)